CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - CSS Warnings (#434)
 - Fix alert banner colors (#442)
+- Fix `SlideOver` scroll background behavior (#420)
```

Comment:
Minor suggestion so [the last version comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first).
```suggestion
- Fix `SlideOver` scroll background behavior (#420)
- CSS Warnings (#434)
- Fix alert banner colors (#442)
```